manifests/default.pp

```diff
@@ -0,0 +1,27 @@
+node 'jenkins-master' {
```

Comment:
this file is only relevant for the vagrant tests, yet it creates a new prominent folder and is placed there. kinda feels it should be rather `vagrant/manifests/default.pp`?

Reply:
Updated. Also moved the Vagrantfile.